src/anomalib/models/components/dinov2/layers/drop_path.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Stochastic depth drop-path implementation used in DINOv2.
+
+This module provides a functional drop-path operation and a corresponding
+nn.Module wrapper. Drop-path (also known as stochastic depth) randomly
+drops entire residual branches during training to improve model robustness.
+"""
+
+from __future__ import annotations
+
+from torch import Tensor, nn
+
+
+def drop_path(x: Tensor, drop_prob: float = 0.0, training: bool = False) -> Tensor:
```

Comment:
```suggestion
def drop_path(x: torch.Tensor, drop_prob: float = 0.0, training: bool = False) -> torch.Tensor:
```

pep and ruff guidelines suggest `torch.Tensor` over `Tensor`
Can you update all the annotations?